web/sites/default/config/webform.webform.report_e_scooter.yml

```diff
@@ -233,19 +237,27 @@ elements: |-
   computed_urgency:
     '#type': webform_computed_twig
     '#title': 'Computed Urgency'
-    '#display_on': none
-    '#template': '{% if data.company_sla == "1" %}Urgent {% endif %}'
+    '#wrapper_attributes':
+      class:
+        - d-none
+    '#display_on': form
+    '#template': "{% if data.report_issue == 'improperly parked' and data.report_blocking == 'Yes' %}URGENT {% endif %}"
+    '#ajax': true
   computed_urgency_text:
     '#type': webform_computed_twig
     '#title': 'Computed Urgency Text'
-    '#display_on': none
+    '#wrapper_attributes':
+      class:
+        - d-none
```

Comment:
Is there a reason these two computed fields are set to display on the form but hidden, instead of `none` like before? Only asking because, when there are ajax fields on the form, it can bog it down a bit because it sends a request and waits for a response from the server whenever the user changes a field value.